### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by Calvinthesneak**
Added section about spawn point colors on map, since I can't seem to find it on the wiki currently....

## Description
Added section about spawn point colors

## Motivation and Context
Spawn point colors used to exist somewhere, people like to know what they mean

## How Has This Been Tested?
Minor text fixes

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/extras/faq.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Common Questions and Answers
 
+## What do the spawn point colors mean?
+
+purple = inactive, not ready to spawn yet.
```
Comment (Arengo) [inline docs/extras/faq.md]: In a FAQ we should write more text than pseudo-code. This is an appearance comment only. How about 
```
Purple: Spawnpoint is inactiv and **not**  about to spawn within the next 5 minutes.
...
```

File: docs/extras/faq.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+blue = about to spawn (light blue 5 mins before spawn, gets darker nearer to spawn time)
+green = spawning
```
Comment (Arengo) [inline docs/extras/faq.md]: Spawnpoint is active, Pokémon has just spawned.

File: docs/extras/faq.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+green becomes yellow then orange which fades to red as the spawn expires.
```
Comment (Arengo) [inline docs/extras/faq.md]: ``Fading from green to orange:``

File: docs/extras/faq.md (hunk 1 above)
Comment (Arengo) [inline docs/extras/faq.md]: In a FAQ we should write more text than pseudo-code. This is an appearance comment only. How about 
```
Purple: Spawnpoint is inactiv and **not**  about to spawn within the next 5 minutes.
...
```

File: docs/extras/faq.md (hunk 2 above)
Comment (Arengo) [inline docs/extras/faq.md]: Spawnpoint is active, Pokémon has just spawned.

File: docs/extras/faq.md (hunk 3 above)
Comment (Arengo) [inline docs/extras/faq.md]: ``Fading from green to orange:``

Comment (Calvinthesneak) [inline docs/extras/faq.md]: I simply grabbed the old documentation I could find.  Thanks for cleaning it up!

Comment (Arengo) [inline docs/extras/faq.md]: I tried to clean it up, but you have to merge this to your PR. Would be nice and probably get merged to develop after that :-)

**Comment by Jesterium** [comment]
Some information is incorrect or omitted.  Please see attached images.

- A **grey** dot represents a spawn point that is more than 5 minutes from spawning
- A **light blue** dot represents a spawn point that will spawn in 5 minutes.  **Light blue** changes to **dark blue** and finally into **purple** just before spawn time.
- A **green** dot represents a fresh spawn.  This will transition to **yellow**, through **orange** and finally **red** (like a stop light) as it is about to despawn.

ex1
ex2

**Comment by SiteEffect** [comment]
@Jesterium sh*t, I had a closer look at the text formatting than at the content itself :-/

**Comment by Calvinthesneak** [comment]
Well then, we'll just copy pasta.....

**Comment by FrostTheFox** [comment]
one more +1 for merge

**Pull request merged**